src/main/java/uk/gov/hmcts/cmc/claimstore/appinsights/AppInsightsEvent.java

```diff
@@ -12,6 +12,7 @@
     CCJ_REQUESTED("CCJ requested"),
     RESPONSE_MORE_TIME_REQUESTED("Response - more time requested"),
     RESPONSE_MORE_TIME_REQUESTED_PAPER("Response - more time requested paper"),
+    RESPONSE_ACCEPTED_SIGN_SETTLEMENT("Response accepted - sign settlement agreement"),
```

Comment:
Would this make more sense?`Claimant response - settlement agreement created`

Reply:
`Offer accepted` is also a claimant response. some thing to differentiate these two would be better. any other option?